charts/rancher-monitoring/v0.0.5/values.yaml

```diff
@@ -317,6 +317,9 @@ prometheus:
     proxy:
       repository: rancher/nginx
       tag: 1.17.4-alpine
+    thanos:
+      repository: improbable/thanos
+      tag: v0.6.0
```

Comment:
[`improbable/thanos` docker image repo](https://hub.docker.com/r/improbable/thanos) is marked as deprecated. Could you please update thanos image with new one??

```
-      repository: improbable/thanos
-      tag: v0.6.0
+      repository: thanosio/thanos
+      tag: v0.9.0
```

@cjellick , as [@drpebcak  mentioned](https://github.com/rancher/system-charts/pull/95#issuecomment-530576997)  Would be required to mirror thanos image in Rancher org??

Reply:
@rawmind0, yes, we should host the image on the Rancher org.